ground/src/main/res/values/strings.xml

```diff
@@ -188,4 +188,8 @@
   <string name="data_submitted_image">Data submitted image</string>
   <string name="data_collection_complete">Data collection complete!</string>
   <string name="data_collection_complete_details">Your data has been saved and will be automatically synced when you’re online.</string>
+
+  <string name="allow_location_confirmation">Allow Location</string>
```

Comment:
Please update based on design in https://github.com/google/ground-android/issues/2658#issuecomment-2402992661. Be sure to use "Sentence case" for headings instead of "Title Case".

Reply:
done